test/test.js

```diff
@@ -235,6 +235,37 @@ describe('Page', function() {
     }));
   });
 
+  describe('Browser.Events.disconnected', function() {
+    it('should emitted when browser gets closed', SX(async function() {
+      const originalBrowser = await puppeteer.launch(defaultBrowserOptions);
+      let isDisconnected = false;
+      originalBrowser.on('disconnected', () => { isDisconnected = true; });
+      await originalBrowser.close();
+      expect(isDisconnected).toBe(true);
+    }));
+    it('should emitted when browser.disconnect() called', SX(async function() {
```

Comment:
Let's unify these three tests into one. This will save us on some time while running tests.

```js
const originalBrowser = await puppeteer.launch(defaultBrowserOptions);
const browserWSEndpoint = originalBrowser.wsEndpoint();
const remoteBrowser1 = await puppeteer.connect({browserWSEndpoint});
const remoteBrowser2 = await puppeteer.connect({browserWSEndpoint});

let disconnectedOriginal = 0;
let disconnectedRemote1 = 0;
let disconnectedRemote2 = 0;
originalBrowser.on('disconnected', () => ++disconnectedOriginal);
remoteBrowser1.on('disconnected', () => ++disconnectedRemote1);
remoteBrowser2.on('disconnected', () => ++disconnectedRemote2);

await remoteBrowser2.disconnect();
expect(disconnectedOriginal).toBe(0);
expect(disconnectedRemote1).toBe(0);
expect(disconnectedRemote2).toBe(1);

await originalBrowser.close();
expect(disconnectedOriginal).toBe(1);
expect(disconnectedRemote1).toBe(1);
expect(disconnectedRemote2).toBe(1);